v2/app/src/main/java/com/google/firebase/example/makeitso/ui/home/HomeScreen.kt

```diff
@@ -0,0 +1,14 @@
+package com.google.firebase.example.makeitso.ui.home
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.hilt.navigation.compose.hiltViewModel
+import kotlinx.serialization.Serializable
+
+@Serializable
+object HomeRoute
+
+@Composable
+fun HomeScreen(viewModel: HomeViewModel = hiltViewModel()) {
+    val uiState = viewModel.uiState.collectAsState()
```

Comment:
`collectAsStateWithLifecycle()` is the latest [recommendation from Android](https://developer.android.com/develop/ui/compose/state#use-other-types-of-state-in-jetpack-compose) - should we use that in this app? Applies throughout. :)